syntax = "proto3";

package fixbug;
option cc_generic_services = true;

message ResultCode 
{
    int32 err_code = 1;
    bytes err_msg = 2;
}

message GetFriendListRequest
{
    uint32 id = 1;
}

message GetFriendListResponse
{
    ResultCode rscode = 1;
    repeated bytes friends = 2;
}


service FriendServiceRpc
{
    rpc GetFriendList(GetFriendListRequest) returns(GetFriendListResponse);
}
